### ISSUE TITLE
Automatischer Darkmode

### ISSUE THREAD

**Issue by math-GH** [heimseiten/contao-dark-mode-backend-bundle#9]
Gibt es Überlegungen den Darkmode anhand der Betriebssystem/Browser-Einstellungen zu erkennen?

Es ist wohl nur 1 Zeile JavaScript: https://stackoverflow.com/questions/56393880/how-do-i-detect-dark-mode-using-javascript

**Comment by akroii** [comment]
Frage würde das auch FOUC beim Seitenwechsel beseitigen?

**Comment by heimseiten** [comment]
Nein. Zur Zeit wird das Farbschema mit JavaScript gesteuert. Das heißt, es greift erst kurz nachdem Seite geladen ist. Ich werde bei Gelegenheit einmal prüfen, wie es ohne FOUC umgesetzt werden könnte.
